src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -81,4 +75,63 @@ static bool TryTestOne<T>(MethodInfo method, Func<T, SharpFuzz.ReadOnlySpanActio
             return TryTestOne<TestOneArray>(method, t => span => t(span.ToArray()));
         }
     }
+
+    class LibFuzzerDotnetTarget
+    {
+        public string AssemblyPath { get; }
+        public string ClassName { get; }
+        public string MethodName { get; }
+
+        public LibFuzzerDotnetTarget(string assemblyPath, string className, string methodName)
+        {
+            AssemblyPath = assemblyPath;
+            ClassName = className;
+            MethodName = methodName;
+        }
+
+        public static LibFuzzerDotnetTarget FromEnvironment()
+        {
+            try {
+                return FromEnvironmentVarDelimited();
+            }
+            catch
+            {}
+
+            try {
+                return FromEnvironmentVars();
+            }
+            catch
+            {}
+
+            throw new Exception("No fuzzing target specified by environment variables");
```

Comment:
It could be useful to include what we're expecting to see in the exception string. I think if the person hit this exception either they forgot the env vars are required or didn't know they were required in the first place. Something like:

`Expected either LIBFUZZER_DOTNET_TARGET=<assembly>:<class>:<method> or LIBFUZZER_DOTNET_TARGET_ASSEMBLY=<assembly>;LIBFUZZER_DOTNET_TARGET_CLASS=<class>;LIBFUZZER_DOTNET_TARGET_METHOD=<method> to be set`

Or even just a link to documentation explaining it.